lua/base46/init.lua

```diff
@@ -50,6 +50,9 @@ M.extend_default_hl = function(highlights)
     for key, value in pairs(polish_hl) do
       if highlights[key] then
         highlights[key] = M.merge_tb(highlights[key], value)
+      else
+        -- add any other highlights from polish_hl
+        highlights[key] = value
```

Comment:
It feels to me like theme authors should be able to extend custom theme highlights to their hearts content.